### DISCUSSION TITLE
2023-07-05 Weekly Meeting 114 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#402] [General]
Weekly Meeting 114 is happening on Wednesday, July 5, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- MSF POG status 
  - <img width="1168" alt="Screenshot 2023-07-05 at 11 38 02 AM" src="https://github.com/omigroup/omigroup/assets/63426722/2394ade1-bf85-489d-a524-531a76f607ee">
  - <img width="1100" alt="Screenshot 2023-07-05 at 11 38 31 AM" src="https://github.com/omigroup/omigroup/assets/63426722/efddc778-273e-44a9-85c7-f61ee6a4ff36">

- Open Collective Update 
  - <img width="1510" alt="Screenshot 2023-07-05 at 11 39 24 AM" src="https://github.com/omigroup/omigroup/assets/63426722/8563fc30-b2f3-463f-8d40-59413d180c01">
  - re: payments. I need help. <img width="1126" alt="Screenshot 2023-07-05 at 11 37 47 AM" src="https://github.com/omigroup/omigroup/assets/63426722/0d3a0ab8-0ed9-4bc4-9bc2-9f7ef9c714b9">
- Jin submitted a prop https://github.com/omigroup/omigroup/issues/400

### UPDATES:

- MSF delegates
- Media group
- Avatar interop group
- GLTF group
- World Hops

**Comment by madjin** [comment]
## Media group


Discussed world building and how it can improve media production, some assets / tools that can be built:
- https://github.com/omigroup/omi-archive/issues/6
- https://github.com/omigroup/omi-archive/issues/5


---

## Avatar Interop Group

Activity on the github: https://github.com/MetaverseStandards/Avatar-Wearables/discussions

Working on converting the knowledge base into knowledge space, 2.0 version

![brave_YBLp9LdRMF](https://github.com/omigroup/omigroup/assets/32600939/9c5bbe68-7f59-4ed6-bb5a-5a9274c9151a)

![brave_ogako0T0He](https://github.com/omigroup/omigroup/assets/32600939/71da14d4-4cf9-4a59-863f-6b1f941aefe2)

## glTF extensions group

Still working on changes with physics shape, spent entire last meeting talking about it
